SPECS/packer/packer.spec

```
@@ -43,7 +43,8 @@ Packer is a tool for building identical machine images for multiple platforms fr
 %build
 tar --no-same-owner -xf %{SOURCE1}
 export GOPATH=%{our_gopath}
-go build -mod=vendor -v -a -o packer
+LD_FLAGS="-X github.com/hashicorp/packer/version.Version=1.8.7 -X github.com/hashicorp/packer/version.VersionPrerelease="
+go build -mod=vendor -v -a -o packer --ldflags="$LD_FLAGS"
```

Although the suggestion from the reference was `--ldflags` and compiler also does not error out, `go help build` suggests to use `-ldflags` instead of `--ldflags`

```suggestion
go build -mod=vendor -v -a -o packer -ldflags="$LD_FLAGS"
```